perfkitbenchmarker/providers/__init__.py

```diff
@@ -51,6 +52,7 @@ def LoadProvider(provider_name):
   Args:
     provider_name: The name of the package whose modules should be loaded.
   """
+  requirements.CheckProviderRequirements(provider_name)
```

Comment:
Can you add a note in `LoadProvider`'s docstring that it checks requirements in addition to loading modules?